docs/code-style.md

```diff
@@ -0,0 +1,132 @@
+# Code style
+
+## JavaScript
+
+We use [ESlint](https://eslint.org/) and [Prettier](https://prettier.io/) to enforce our JavaScript code style. 
+Running `make fix` after making any changes will fix most things.
+
+There are some styles the linter can't pick up. If you are unsure of anything, [AirBnB's style
+guide](https://github.com/airbnb/javascript) is a good place to look.
+
+## Components
+
+We use [React](https://reactjs.org/) for our components, and [Emotion](https://emotion.sh/) for our CSS in JS.
+
+- Use Emotion's [`css`](https://emotion.sh/docs/css) function to build a class name from styles
+
+> Why? It decouples our CSS declaration from specific components
+
+```js
+// bad
+const MyComponent = styled('div')`
+    color: red;
+`;
+render(
+    <MyComponent />
+);
+
+// bad
+render(
+    <div css={`
+        color: red;
+    `} />
+);
+
+// good
+const myComponent = css`
+    color: red;
+`;
+render(
+    <div className={myComponent} />
+);
+```
+
+- Extract CSS into a variable rather than defining it inline in a component
```

Comment:
Is it okay to make these headings rather than list items? Mostly because they don't stand out very much with Github's current styling at least.

Reply:
Yeah, I'll try it out! I didn't want them to be too big and distracting

Reply:
It looks okay!

Added bonus: anchor links!